cwltool/main.py

```diff
@@ -149,6 +152,15 @@ def arg_parser():  # type: () -> argparse.ArgumentParser
     exgroup.add_argument("--quiet", action="store_true", help="Only print warnings and errors.")
     exgroup.add_argument("--debug", action="store_true", help="Print even more logging")
 
+    # help="Dependency resolver configuration file describing how to adapt 'SoftwareRequirement' packages to current system."
+    parser.add_argument("--beta-dependency-resolvers-configuration", default=None, help=argparse.SUPPRESS)
```

Comment:
Here-ish https://docs.galaxyproject.org/en/latest/admin/dependency_resolvers.html#dependency-resolvers-in-galaxy but the Galaxy docs reference Galaxy requirements and don't mention the file can be YAML like in the examples I've included here or using URIs to disambiguate packages or the local mapping stuff that makes the environment modules work more usable. So I guess the real answer is scattered across a bunch of PRs. Where do you want the cwltool documentation? Should I add it to a new section of the README or create a new file?